ui/src/app/shared/status/status.icon.html

```diff
@@ -6,5 +6,5 @@
     <i class="remove red icon" *ngSwitchCase="'Fail'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Disabled'"></i>
     <i class="ban grey icon" *ngSwitchCase="'Skipped'"></i>
-    <i class="wait orange icon" *ngSwitchCase="'Waiting'"></i>
+    <i class="wait blue icon" *ngSwitchCase="'Waiting'"></i>
```

Comment:
I think your pb is with CLI. But in WebUI i d'ont think it's a bad idea. 
Waiting: blue clock icon
Building: blue loader